app/models/alert_date.py

```diff
@@ -60,7 +66,7 @@ def as_lang(self, lang='en-GB'):
     @property
     def as_alert_lang(self, lang='en-GB'):
         dt = self._local_datetime
-        return f'{dt:%A} {dt.day} {dt:%B} {dt:%Y}, at {dt:%-I%p}'
+        return f'{dt:%A} {dt.day} {dt:%B} {dt:%Y} at {dt:%-I%p}'
```

Comment:
Is the only difference between this and the function above that one has AM in capitals and the other has AM in lowercase?

If that's the case I think we may as well just settle on one format and use it for all of them.

Reply:
This particular line just removes the comma - in the soft launch on the 19th Mar, it was suggested that the announcement banner had too many commas and didn't scan well.

Regarding the upper/lower case methods: As the uppercase has been the only option until now, I assume that this is in line with the GOV.UK style guide.  This looked a little jarring in the banner and announcement headers, which is why I implemented a lowercase option. I think we should still keep both options.

Reply:
https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates

I don't think we should have 2 different options, we should have one and stick with it.  We can check with the content team to decide which one.

Reply:
Mehmet confirmed that "gov.uk content" team were happy with the lowercase am/pm